kyo-core/shared/src/test/scala/kyoTest/KyoTest.scala

```diff
@@ -7,6 +7,7 @@ import org.scalatest.concurrent.ScalaFutures.*
 import org.scalatest.freespec.AsyncFreeSpec
 import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
+import scala.language.implicitConversions
```

Comment:
There's an implicit conversion in this class from `T` to `Future[T]`. I think it's ok to keep it for now and remove it later if we need to.